src/identity/actions/thunks/index.ts

```diff
@@ -0,0 +1,121 @@
+import {GetState} from 'src/types'
+import {Dispatch} from 'react'
+
+// Actions
+import {
+  setQuartzMe,
+  setQuartzMeStatus,
+  Actions as MeActions,
+} from 'src/me/actions/creators'
+import {
+  setCurrentOrgDetails,
+  setCurrentBillingProvider,
+  setQuartzIdentity,
+  setQuartzIdentityStatus,
+  Actions as IdentityActions,
+} from 'src/identity/actions/creators'
+
+// Types
+import {RemoteDataState} from 'src/types'
+enum RetrievalError {
+  accountIdentity = 'Unable to retrieve account ID from user identity.',
+  orgIdentity = 'Unable to retrieve organization ID from user identity.',
+}
+
+// Utilities
+import {
+  convertIdentityToMe,
+  fetchQuartzIdentity,
+  fetchAccountDetails,
+  fetchOrgDetails,
+} from 'src/identity/apis/auth'
+
+// Retrieves user's quartz identity from /quartz/identity, and stores it in state.identity.
```

Comment:
I'm kind of ambivalent about this comment. It basically repeats what the code is doing. I think we should use comments to explain why the code is doing what it's doing, not what the code is doing, since the code explains what it's doing.

Reply:
Helpful, thanks. I'm removing comments in the PR to the extent they're just describing something that can pretty easily be figured out by just reading the function.